owners.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@orlandohohmeier  : agree. We need to converge on a owners format. This will be starting version to make sure that we adhere a concept of "owners". Right now we have a flat ownership structure, and there is a desire to come up with more granular/hierarchical ownership.  In the future versions, we will adopt one of Kubernetes/Chromium/Android/Google project style owners. Right now, it is just json keys with values for slack handles for slack notifications.